R/session/vsc.R

```diff
@@ -428,8 +465,24 @@ if (use_httpgd && "httpgd" %in% .packages(all.available = TRUE)) {
           dev_args <- getOption("vsc.dev.args")
           do.call(png, c(list(filename = plot_file), dev_args))
           on.exit({
+            cur_dev <- dev.cur()
             dev.off()
+            created_devices <<- created_devices[created_devices != cur_dev]
             cat(get_timestamp(), file = plot_lock_file)
+            if (!is.na(request_tcp_connection)) {
+              tryCatch({
+                  plot_file_content <- readr::read_file_raw(plot_file)
```

Comment:
Does `readr::read_file_raw()` behave differently from `readBin(file, file.size(file))`?

Reply:
Maybe `readBin(file, file.size(file))` is good enough so that we don't have to introduce `readr` as a new dependency here.

Reply:
> Maybe `readBin(file, file.size(file))` is good enough so that we don't have to introduce `readr` as a new dependency.

You need to specify the correct encoding parameters, and I'm unable to make this work with jsonlite::base54_enc function with `readBin`.